HomeWork29/HomeWork29/NetworkService.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  NetworkService.swift
+//  HomeWork29
+//
+//  Created by Stanislav Sakhipov on 9/23/24.
+//
+
+import Foundation
+
+class NetworkService {
+    static let users = "https://jsonplaceholder.typicode.com/users"
+    
+    func loadUsers(completion: @escaping ([User]) -> Void) {
```

Comment:
тут можно сделать что-то типа
```suggestion
    func loadUsers(completion: @escaping (Result<[User], MyError>) -> Void) {
```

Reply:
где MyError какая-то ошибка которую можно обработать потом выше)